src/scripts/OSFramework/OSUI/Helper/SVG.ts

```diff
@@ -0,0 +1,25 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace OSFramework.OSUI.Helper {
+	// eslint-disable-next-line @typescript-eslint/naming-convention
+	export abstract class SVG {
+		/**
+		 * Function that validates if a given svgString is a valid SVG
+		 *
+		 * @param url
+		 * @memberof OSFramework.Helper.URL
+		 */
+		public static IsValid(svgString: string): boolean {
+			const parser = new DOMParser();
+			try {
+				const doc = parser.parseFromString(svgString, 'image/svg+xml');
```

Comment:
In order to keep consistency, I suggest to Enum the text on method.
- 'image/svg+xml'
- 'parsererror'
- 'svg'

Reply:
done